### DISCUSSION TITLE
Setting gutter color with CSS

### DISCUSSION THREAD

**Discussion by jalanh11640** [ryanluker/vscode-coverage-gutters#460] [Q&A]
I see that setting the gutter color with CCS was introduced in the last release, thanks for that. Where can I find an example of how to do this?

Thanks

**Comment by mattseddon** [comment]
The settings that you are looking for are `coverage-gutters.highlightlight, coverage-gutters.highlightdark, coverage-gutters.noHighlightDark, coverage-gutters.noHighlightLight, coverage-gutters.partialHighlightDark, coverage-gutters.partialHighlightLight`.

Below I've set covered lines to pink (because I hate my eyes):

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/45ffaa78-7392-4fc0-9bed-9c30cdcb45b3">

The defaults should be shown in your IDE if you access the settings UI via the command palette:

<img width="755" alt="image" src="https://github.com/user-attachments/assets/b0525826-3eeb-493c-a677-ee3d94b34e97">

and then search for coverage gutters:

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/4b2a0cff-962c-4e0d-8ade-c315d3c83026">

Hope this helps.

**Reply by jalanh11640** [reply]
Thanks for replying!

I'm not seeing any change in the coverage colors.

Here's what I have in my settings.json file:
```
    "coverage-gutters.coverageFileNames": [
        "**/coverage.info"
    ],
    "coverage-gutters.highlightdark": "rgba(232, 245, 39, 0.8)",
    "coverage-gutters.highlightlight": "rgba(232, 245, 39, 0.8)",
    "coverage-gutters.showLineCoverage": true
```

This should be a yellowish color.  Instead, this is what I see:
![image](https://github.com/user-attachments/assets/b220848e-a5ca-49da-8bb6-bdd5ef3ac748)

I've tried the same value for noHighlight* and partialHighlight* with no change in colors.

I'm running vscode 1.95.1 on an Apple M2 Mac Mini with macOS 15.2 (beta).  The Coverage Gutters extension is v2.12.0 and running inside a docker container.

Is there something I'm missing?

Thanks,
Jeff

**Comment by mattseddon** [comment]
Could have something to do with running inside of a docker container as this works for me:

https://github.com/user-attachments/assets/7e00f1dc-0279-410e-864e-960c94b9f5d3

Maybe try setting the value before starting the container.

**Reply by jalanh11640** [reply]
I figured out the issue. I had inadvertently added the lines to the .vscode/settings.json instead of my user settings.json.  When I moved the settings to my user settings.json I see the new colors!!

Thank you SO much!

**Comment by ryanluker** [comment]
@mattseddon Many thanks for assisting here! Nice to hear you got it working @jalanh11640 !
